### PR TITLE
Add Protobuf templates

### LINKED ISSUE #3769: Add templates for IceRPC + Protobuf
We need the Protobuf equivalent of the IceRPC + Slice templates.

### PR DESCRIPTION
**Pull request by pepone**
Fixes #3769  and #3776

### CODE REVIEW

File: src/IceRpc.Templates/Templates/IceRpc-Protobuf-Client/proto/greeter.proto [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// Copyright (c) ZeroC, Inc.
+
+syntax = "proto3";
+
+package visitor_center;
+option csharp_namespace = "IceRpc_Protobuf_Client";
+
+// The protobuf equivalent of Slice interface Greeter in the Greeter example.
```
Comment (bernardnormier) [inline src/IceRpc.Templates/Templates/IceRpc-Protobuf-Client/proto/greeter.proto:8]: This is not a good description. Protobuf examples / templates should not reference Slice.

File: src/IceRpc.Templates/Templates/IceRpc-Protobuf-DI-Client/proto/greeter.proto [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// Copyright (c) ZeroC, Inc.
+
+syntax = "proto3";
+
+package visitor_center;
+option csharp_namespace = "IceRpc_Protobuf_DI_Client";
+
+// The protobuf equivalent of Slice interface Greeter in the Greeter example.
```
Comment (bernardnormier) [inline src/IceRpc.Templates/Templates/IceRpc-Protobuf-DI-Client/proto/greeter.proto:8]: Needs to be fixed in all proto files.

**Review by bernardnormier — APPROVED** [review]
Looks good, except we need to remove / replace all references to Slice.

File: src/IceRpc.Templates/Templates/IceRpc-Protobuf-DI-Client/.template.config/template.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
+{
+    "$schema":"http://json.schemastore.org/template",
+    "author":"ZeroC",
+    "classifications":[
```
Comment (externl) [inline src/IceRpc.Templates/Templates/IceRpc-Protobuf-DI-Client/.template.config/template.json:4]: is there a DI classification we can/should add here?

**Review by externl — APPROVED** [review]

File: src/IceRpc.Templates/Templates/IceRpc-Protobuf-DI-Client/.template.config/template.json (hunk 3 above)
Comment (pepone) [reply src/IceRpc.Templates/Templates/IceRpc-Protobuf-DI-Client/.template.config/template.json:4]: I don't see anything in the default templates:
https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new-sdk-templates

**Pull request merged**